hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/StoreFileWriter.java

```diff
@@ -181,7 +197,10 @@ public long getPos() throws IOException {
    */
   public void appendMetadata(final long maxSequenceId, final boolean majorCompaction)
     throws IOException {
-    appendMetadata(maxSequenceId, majorCompaction, Collections.emptySet());
+    liveFileWriter.appendMetadata(maxSequenceId, majorCompaction);
```

Comment:
OK, checked the other multi file writer implementation, found HBASE-15400, where we explained why we use the same maxSequenceId for multiple genedated store files. So at least this is not a critical problem.